scripts/dapper-image

```diff
@@ -23,8 +23,15 @@ DAPPER_BASE_IMAGE=${REPO}/dapper-base:${TAG}
 UPSTREAM_REMOTE=$(git remote -v | grep submariner-io/submariner | awk '{ print $1 }' | head -n 1)
 CHANGED_FILES_LOCAL=$(git diff --name-only remotes/$UPSTREAM_REMOTE/master)
 
+# Detect when we are on a merged patch
+LAST_COMMIT_LOCAL=$(git rev-parse HEAD)
+LAST_COMMIT_MASTER=$(git rev-parse remotes/$UPSTREAM_REMOTE/master)
+
+
+
 if [[ "${CHANGED_FILES_PR[@]}" =~ "package/Dockerfile.dapper-base" ]] ||
-   [[ "${CHANGED_FILES_LOCAL[@]}" =~ "package/Dockerfile.dapper-base" ]] ; then
+   [[ "${CHANGED_FILES_LOCAL[@]}" =~ "package/Dockerfile.dapper-base" ]] ||
+   [[ "${LAST_COMMIT_LOCAL}" == "${LAST_COMMIT_MASTER}" ]]; then
```

Comment:
Wouldn't this rebuild it for each merge, no matter if the dapper was modified or not?

Reply:
That seems to be what he wants, according to the commit message. +1 for clarification.

Reply:
Yeah but this AFAICT will rebuild on each time the last commit is a merge, whether the PR changed the base image or not, which I'm not sure is desired since it'll generate these images a whole lot more but most of them will be unchanged

Reply:
I think it's fine :) we're hitting this again.

It's an ok cost to pay for the meanwhile, we can optimize this later ... the big churn of builds is PRs, not merges.